docs/src/components/ComponentStructure/index.tsx

```diff
@@ -0,0 +1,242 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { Grid, Text, useRandomIdSeed, mediaQueries as mq } from "@kiwicom/orbit-components";
+
+import docsTheme from "../../theme";
+import { KEY_CODE } from "../../consts";
+
+// https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role
+// https://dev.to/eevajonnapanula/keyboard-accessible-tabs-with-react-5ch4
+
+const StyledTabList = styled.div`
+  display: flex;
+  gap: 4px;
+  padding: 0 20px;
+`;
+StyledTabList.defaultProps = { theme: docsTheme };
+
+const StyledTab = styled.button<{ active: boolean }>`
+  ${({ theme, active }) => css`
+    padding: 14px ${theme.orbit.spaceMedium};
+    margin-bottom: -1px;
+    border-top-left-radius: ${theme.orbit.borderRadiusLarge};
+    border-top-right-radius: ${theme.orbit.borderRadiusLarge};
+    border: 1px solid transparent;
+    border-bottom: none;
+    ${active &&
+    css`
+      background: ${theme.orbit.paletteWhite};
+      border-color: ${theme.orbit.paletteCloudDark};
+    `};
+  `};
+`;
+StyledTab.defaultProps = { theme: docsTheme };
+
+const StyledPanel = styled.div`
+  ${({ theme }) => css`
+    border: 1px solid ${theme.orbit.paletteCloudDark};
+    border-radius: ${theme.orbit.borderRadiusLarge};
+  `};
+`;
+StyledPanel.defaultProps = { theme: docsTheme };
+
+const StyledImageContainer = styled.div`
+  ${({ theme }) => css`
+    display: grid;
+    place-content: center;
+    padding: ${theme.orbit.spaceXXLarge};
+  `};
+`;
+StyledImageContainer.defaultProps = { theme: docsTheme };
+
+const StyledComponentParts = styled.div`
+  ${({ theme }) => css`
+    display: grid;
```

Comment:
I think you don't need that

Reply:
It's useful in vertically centering the list of parts when the example is taller than the list of parts.